values.yaml

```diff
@@ -147,3 +162,46 @@ extraVolumes: []
 #        - key: cloudfront.pem
 #          path: cloudfront.pem
 #          mode: 511
+
+## Configuration to enable the metrics endpoint of the registry and provide a
+## service port for it
+##
+metrics:
+  enabled: true
+  ## The name of the service resources port for the metrics interface
+  servicePortName: debug
+  ## The service port of the services clusterIP
+  servicePort: 5001
+
+  ## The container port name of the registry pods
+  targetPortName: debug
```

Comment:
I think it's probably okay to be opinionated about what these port names are, rather than making them configurable. Besides, I'm pretty sure the port names are just an implementation detail anyway with no impact outside of how this chart internally "glues" the service/deployment/pod together.